core/server/apps/subscribers/lib/helpers/subscribe_form.js

```diff
@@ -43,6 +43,7 @@ subscribeScript =
 subscribe_form = function (options) {
     var root = options.data.root,
         data = _.merge({}, options.hash, _.pick(root, params), {
+            // @TODO: shall we use `urlJoin` here instead of `path.join`?
```

Comment:
Quite possibly.

Path should be for anything to do with fs, url for anything accessed via HTTP